automates/model_assembly/networks.py

```diff
@@ -1433,15 +1499,25 @@ def from_json(cls, json_path):
         H = [HyperEdge.from_dict(h, ALL_NODES) for h in data["hyper_edges"]]
 
         # TODO: fix this to actually gather typedefs
```

Comment:
I think this todo is resolved now?

Reply:
Yes it is! Thank you for catching that. Deleted!